posthog/api/error_tracking.py

```diff
@@ -333,6 +335,65 @@ def batch_get(self, request, **kwargs):
         return Response({"results": serializer.data})
 
 
+class ErrorTrackingReleaseSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ErrorTrackingRelease
+        fields = ["id", "hash_id", "team_id", "created_at", "metadata"]
+        read_only_fields = ["team_id"]
+
+
+class ErrorTrackingReleaseViewSet(TeamAndOrgViewSetMixin, viewsets.ModelViewSet):
+    scope_object = "error_tracking"
+    queryset = ErrorTrackingRelease.objects.all()
+    serializer_class = ErrorTrackingReleaseSerializer
+
+    def safely_get_queryset(self, queryset):
+        queryset = queryset.filter(team_id=self.team.id)
+
+        return queryset
+
+    def validate_hash_id(self, hash_id: str, assert_new: bool) -> str:
+        if len(hash_id) > 128:
+            raise ValueError("Hash id length cannot exceed 128 bytes")
+
+        if assert_new and ErrorTrackingRelease.objects.filter(team=self.team, hash_id=hash_id).exists():
+            raise ValueError(f"Hash id {hash_id} already in use")
+
+        return hash_id
+
+    def update(self, request, *args, **kwargs) -> Response:
+        release = self.get_object()
+
+        metadata = request.data.get("metadata")
+        hash_id = str(request.data.get("hash_id"))
+
+        if metadata:
+            release.metadata = metadata
+
+        if hash_id and hash_id != release.hash_id:
+            hash_id = self.validate_hash_id(hash_id, True)
+            release.hash_id = hash_id
+
+        release.save()
+        return Response({"ok": True}, status=status.HTTP_204_NO_CONTENT)
+
+    def create(self, request, *args, **kwargs) -> Response:
+        id = UUIDT()  # We use this in the hash if one isn't set, and also as the id of the model
+        metadata = request.data.get("metadata")
+        hash_id = str(request.data.get("hash_id")) or str(id)
```

Comment:
```suggestion
        hash_id = str(request.data.get("hash_id") or id)
```
Otherwise you will get 'None' as a hash_id, right ?

Reply:
Ah, yes, whoops